### PR TITLE
Gym website 

### PR DESCRIPTION
**Pull request by jay-2000**
created a gym website #99

### CODE REVIEW

File: MyGym Website/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# MyGym Fitness website
+This website contains frontend for he gym website, and technologies used are 
+* HTML    
+ 
+* CSS
+
+* JS(BOOTSTRAP)
+
+## some output images
```
Comment (chicken-biryani) [inline MyGym Website/README.md:9]: Change wordings "Some output images" to Output

File: MyGym Website/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+![output1](/images/output1.jpg)
+
+![output2](/images/output2.jpg)
+
+![output2](/images/output3.jpg)
```
Comment (chicken-biryani) [inline MyGym Website/README.md:15]: The images are not working

File: MyGym Website/js/custom.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            scrollTop: $($anchor.attr('href')).offset().top - 49
+        }, 1000);
+        event.preventDefault();
+      });
+    });    
+  });
+
+
+    
+
```
Comment (chicken-biryani) [inline MyGym Website/js/custom.js:31]: Use a JS beautifier to increase the readability.
Here use this link -- https://codebeautify.org/jsviewer

**Comment by chicken-biryani** [comment]
@harshita2216 @Ayushparikh-code can you also review this, please?

File: MyGym Website/README.md (hunk 2 above)
Comment (jay-2000) [reply MyGym Website/README.md:15]: ok, will resolve the problem

File: MyGym Website/js/custom.js (hunk 3 above)
Comment (jay-2000) [reply MyGym Website/js/custom.js:31]: sure!!

**Review by Ayushparikh-code — COMMENTED** [review]
Code looks good to me @jay-2000  just do the changes suggested by @chicken-biryani

**Comment by harshita2216** [comment]
> @harshita2216 @Ayushparikh-code can you also review this, please?

Sure. Looking into it once again before merging.

**Review by Ayushparikh-code — APPROVED** [review]
ok LGTM

**Pull request merged**